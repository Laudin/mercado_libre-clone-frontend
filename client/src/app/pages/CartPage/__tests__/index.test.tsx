import * as React from 'react'
import { createRenderer } from 'react-test-renderer/shallow'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import { CartPage } from '../index'

const shallowRenderer = createRenderer()

describe('<CartPage />', () => {
   test('should render', () => {
      shallowRenderer.render(<CartPage />);
      const rendererOutput = shallowRenderer.getRenderOutput();
      expect(rendererOutput).toMatchSnapshot()
   })
})

//test styled-components
/* it('works fine', () => {
   const page = renderer.create(<HomePage />).toJSON()

}) */