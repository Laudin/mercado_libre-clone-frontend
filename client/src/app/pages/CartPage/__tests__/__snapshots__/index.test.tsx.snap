// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> should render 1`] = `
<React.Fragment>
  <Helmet
    defer={true}
    encodeSpecialCharacters={true}
    prioritizeSeoTags={false}
  >
    <title>
      HomePage
    </title>
    <meta
      content="A Boilerplate application homepage"
      name="description"
    />
  </Helmet>
  <HomePage__Container>
    <Slider />
    <HomePage__ContentWrapper>
      <HomePage__Content>
        <ProductsList />
      </HomePage__Content>
    </HomePage__ContentWrapper>
  </HomePage__Container>
</React.Fragment>
`;
